.claude/skills/handle-exception/SKILL.md

```
@@ -0,0 +1,47 @@
+---
+
+# 예외 처리 및 ErrorCode 작성 스킬
+
+## 포맷
+```java
+NAME(HttpStatus.STATUS, "DOMAIN_x001", "~입니다."),
+```
+허용 상태: 400 / 401 / 403 / 404 / 409
+
+## 코드 범위
+| 범위 | 상태 |
```

markdownlint-cli2 (0.22.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)